lib/documentation/index.js

```diff
@@ -10,7 +11,9 @@ const mkdirp = require('mkdirp');
 
 const entries = api['symbols'];
 const compiledHandlebars = Handlebars.compile(template);
+const compiledSinceTemplate = Handlebars.compile(sinceTemplate);
 const linkMatcher = /{@link(\s)(\w+)\s*}/gi;
+const sinceMarker = "<!--since_tag_marker-->";
```

Comment:
Feel free to suggest different way to display the since version, I did it with special comment, replaced build time.

Reply:
Looks simple enough for now so you can keep it, but if we start replacing more things, we should consider something like EJS.